caesars-palace/blocks/header/header.js

```diff
@@ -129,6 +131,54 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+const showMore = (nav, maxItemsDesktop) => {
```

Comment:
Something that _may_ help performance is to group element creations and DOM change (instead of having it interleaved with the lines that add classes). I know it makes the code harder to understand, but it makes it easier for the browser to optimize. @swetabar Do you want to try it out to see if it changes perf?

Reply:
I think the low scores were a result of the main page - multiple carousels / other elements on that page. I'll probably leave this as is, for now.